javaparser-core/src/main/java/com/github/javaparser/ast/body/MethodDeclaration.java

```diff
@@ -363,8 +363,8 @@ public String getDeclarationAsString(boolean includingModifiers, boolean includi
 
     @Override
     public JavadocComment getJavadocComment() {
-        if (getComment() instanceof JavadocComment) {
-            return (JavadocComment) getComment();
+        if (getComment().isPresent() && getComment().get() instanceof JavadocComment) {
```

Comment:
```
 if (getComment().isPresent() && getComment().get() instanceof JavadocComment) {
           return (JavadocComment) getComment().get();
 }
```
We are doing a lot of same stuff here in each node. Let's introduce a utility method?